src/App.tsx

```diff
@@ -119,23 +125,54 @@ function App(): ReactElement {
   // UTILITY METHODS /////////////////////////////////////////////////////////////
 
   /**
-   * Get a set of URL parameters that represent the current collection, dataset, feature, track,
-   * and frame information. (Convenience wrapper for `urlUtils.getUrlParams`.)
+   * Get a URL query string representing the current collection, dataset, feature, track,
+   * and frame information.
    */
   const getUrlParams = useCallback((): string => {
     let datasetParam: string | undefined = datasetKey;
-    if (!collection?.url) {
-      // The collection has no source file; use the dataset URL instead
+    let collectionParam = collection?.url;
+    // If there is no collection loaded (e.g. a single dataset URL was loaded),
+    // use the dataset URL instead.
+    if (collectionParam === null) {
       datasetParam = dataset?.manifestUrl;
+      collectionParam = undefined;
+    }
+    // If the collection is the default collection path, don't include it in the URL.
```

Comment:
I think this should just always be a full url.  There isn't really a default collection anymore, is there?